dockerfiles/jaeger/agent/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This Dockerfile tracks jaegertracing/jaeger-agent, but includes more debugging tools and runs as a
+# non-root user. It requires JAEGER_VERSION to be set as an argument to build.
+ARG JAEGER_VERSION
+FROM jaegertracing/jaeger-agent:$JAEGER_VERSION as base
+
+FROM alpine:3.11.5
+RUN apk --no-cache bash curl
```

Comment:
Are these Dockerfiles ONLY going to be used by deploy-sourcegraph, or are they also going to be used by deploy-sourcegraph-docker?

If the latter, these should all be moved into https://github.com/sourcegraph/sourcegraph/tree/master/docker-images and follow the conventions there.

This is also extremely important because I want us to get all Docker images versioned alongside Sourcegraph (`3.14.1` instead of `<arbitrary>` tags).